controller/login_test.go

```diff
@@ -148,6 +162,68 @@ func (rest *TestLoginREST) TestLinkIdPWithTokenRedirects() {
 	test.LinkLoginTemporaryRedirect(t, svc.Context, svc, ctrl, nil, nil)
 }
 
+func (rest *TestLoginREST) TestResourceRequestPayload() {
+	t := rest.T()
+	resource.Require(t, resource.Database)
```

Comment:
`accessToken, err := GenerateUserToken(service.Context, tokenEndpoint, rest.configuration, rest.configuration.GetKeycloakTestUserName(), rest.configuration.GetKeycloakTestUserSecret())`
does call Keycloa, doesn't it?